tvm/python/tvm/topi/adreno/conv2d_nhwc.py

```diff
@@ -281,17 +333,31 @@ def copy_to_texture(stage):
 
     s[latest].compute_root()
 
-    axes = s[latest].op.axis
-    fused = s[latest].fuse(*axes[:-1])
     N, OH, OW, OC = get_const_tuple(latest.shape)
 
-    if OC < 32:
-        block, thread = s[latest].split(fused, factor=32)
-        s[latest].bind(block, te.thread_axis("blockIdx.x"))
-        s[latest].bind(thread, te.thread_axis("threadIdx.x"))
+    if OC % 4 == 0:
+        n, oh, ow, oc = s[latest].op.axis
+        occ, ocb = s[latest].split(oc, factor=4)
+        s[latest].reorder(n, oh, ow, occ, ocb)
+        s[latest].vectorize(ocb)
+        fused = s[latest].fuse(n, oh, ow, occ)
+
+        ftc = N * OH * OW * OC / 4
+        div = getDiv(ftc, 128)
```

Comment:
could you please provide comment on the reason for choosing 128? Is it workgroup or warp size?

Reply:
see above comment